karma.conf.js

```diff
@@ -24,7 +24,8 @@ module.exports = function(config) {
 			// source files, that you wanna generate coverage for
 			// do not include tests or libraries
 			// (these files will be instrumented by Istanbul)
-			'app/scripts/**/*.js': sourcePreprocessors
+			'app/scripts/**/*.js': sourcePreprocessors,
+			'test/spec/**/*.js': sourcePreprocessors
```

Comment:
Regarding ignore directives they are for difficult code paths on the tested code, not tests code necessary, that is the reason you don't see mentions in Istanbul, both practices are related to coverage in any platform and technology.

Tried and can't find where I read about, will give it another try and let you know...